crates/gitbutler-repo/src/rebase.rs

```diff
@@ -419,11 +421,237 @@ fn resolve_index(
 
 #[cfg(test)]
 mod test {
+    use std::path::Path;
```

Comment:
I highly recommend to put tests for the public API into integration-`./tests` next to `./src`. That way, the source file won't be cluttered up and one is assured to test like one would as consumer of the library.